airgun/views/host.py

```diff
@@ -199,7 +199,7 @@ def is_displayed(self):
 class HostsView(BaseLoggedInView, SearchableViewMixin):
     title = Text("//h1[normalize-space(.)='Hosts']")
     export = Text(".//a[contains(@class, 'btn')][contains(@href, 'hosts.csv')]")
-    new = Text("//a[contains(normalize-space(.),'Create Host')]")
+    new = Text(".//div[@id='rails-app-content']//a[contains(normalize-space(.),'Create Host')]")
```

Comment:
The previous locator was matching also "Create Host" from the nav bar that was actually not visible, this will make sure that the only locator on the main page is matched. Tested for empty host view and also with some hosts.